plugins/terraform/layer0/data_source_layer0_environment.go

```diff
@@ -77,9 +73,9 @@ func dataSourceLayer0EnvironmentRead(d *schema.ResourceData, meta interface{}) e
 	d.SetId(environment.EnvironmentID)
 	d.Set("name", environment.EnvironmentName)
 	d.Set("instance_type", environment.InstanceType)
-	d.Set("min_scale", environment.MinScale)
-	d.Set("current_scale", environment.CurrentScale)
-	d.Set("max_scale", environment.MaxScale)
+	d.Set("environment_type", environment.EnvironmentType)
+	d.Set("environment", environment.EnvironmentType)
```

Comment:
line 77 seems like an accidental copy+paste and needs to be deleted `d.Set("environment", environment.EnvironmentType)`

Reply:
✅